.eslintrc.js

```diff
@@ -1,6 +1,35 @@
 const sharedConfigs = require('superdesk-code-style');
 
 module.exports = Object.assign({}, sharedConfigs, {
+    rules: Object.assign(sharedConfigs.rules, {
```

Comment:
Should include/discuss all this rules in https://github.com/superdesk/superdesk-client-code-style?

Reply:
I want to finish ansa stuff before going on holiday next week, but you can do it without me.